bundles/statistics/statsgrid/FlyoutManager.js

```diff
@@ -89,7 +89,13 @@ Oskari.clazz.define('Oskari.statistics.statsgrid.FlyoutManager', function (insta
             return;
         }
         if (type === 'search') {
-            this.searchHandler.getController().toggleSearchFlyout(true);
+            this.searchHandler.getController().toggleSearchFlyout(true, () => this.trigger('hide', type));
+            this.hide('table');
```

Comment:
Does this mean that both flyouts can't be open at the same time? Currently they can be.

Reply:
Yes, did this because the react flyouts will open on top of each other currently.